cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -27,9 +27,11 @@ class TLMOptions(TypedDict):
 
     Attributes:
         max_tokens (int): the maximum number of tokens to generate in the TLM response
+        use_self_reflection (bool): Optional flag to turn on/off asking the TLM self_reflection questions
```

Comment:
FYI -- the correct way to type this will be by defining the args as `Required` or `NotRequired`, not as `Optional`. `Optional` connotes that the value may be `None`, while `NotRequired` connotes that the key may not be present in the dict.

Make sure to use `TypedDict` from `typing_extensions` to implement this -- this isn't a built-in until 3.11.

[Relevant PEP](https://peps.python.org/pep-0655/)